sql_generators/glean_usage/common.py

```diff
@@ -104,7 +104,11 @@ def table_names_from_baseline(baseline_table, include_project_id=True):
 def referenced_table_exists(view_sql):
     """Dry run the given view SQL to see if its referent exists."""
     dryrun = DryRun("foo/bar/view.sql", content=view_sql)
-    return 404 not in [e.get("code") for e in dryrun.errors()]
+    # 403 is returned if referenced dataset doesn't exist
```

Comment:
I'd think 403 would also be returned if whatever account is doing the dry-running doesn't have permission to access a referenced table/view, but in that scenario I guess we also wouldn't want to generate Glean usage views.